General.xcodeproj/xcshareddata/xcschemes/GeneralIOs.xcscheme

```diff
@@ -52,7 +52,7 @@
       </BuildableProductRunnable>
       <CommandLineArguments>
          <CommandLineArgument
-            argument = "bootstrap -n Sri-iOS -u Uml/sri.uml"
+            argument = "bootstrap -n Sri-iOS "
```

Comment:
is it ok to use `Sri-iOS` here?

Reply:
Yes, the name of the project is a mandatory argument, otherwise bootstrap command will not be executed. For example, the project name is Sri-iOS.